packages/plugin-commands-init/src/init.ts

```diff
@@ -40,8 +40,11 @@ export async function handler (
   if (fs.existsSync(manifestPath)) {
     throw new PnpmError('PACKAGE_JSON_EXISTS', 'package.json already exists')
   }
+  const response = await fetch('https://registry.npmjs.org/pnpm/latest')
+  const version = (await response.json() as any).version // eslint-disable-line
   const manifest = {
     name: path.basename(process.cwd()),
+    packageManager: `pnpm@${version}`,
```

Comment:
I don't think you want this field to be in this location.

Reply:
Should it be latest or should it be the current version of the CLI?

Reply:
> I don't think you want this field to be in this location.

I'm not sure where to put it, so I've moved it above the script.

Reply:
> Should it be latest or should it be the current version of the CLI?

Done.